workflows/striking-product-images/README.md

```diff
@@ -0,0 +1,56 @@
+# Generate Striking Product Images
+
+<div align="center">
+    <img src="images/striking-product-images-full.jpeg" alt="Product Image Generator">
+</div>
+
+## Overview
+Need eye-catching product images that pop? With this AI flow, you can turn any ordinary photo into a stunning, professional-looking visual in seconds. Perfect for e-commerce, marketing, or product showcases. Say goodbye to boring backgrounds and hello to sleek, polished images that stand out! Let the AI handle it all—quick, easy, and powerful.
```

Comment:
same. we should change ChatGPT system prompt to get Overview text :) @emreyrtsvn 

let's write a bit more technical instead of marketing and fancy words.